packages/expo-image-manipulator/src/ImageManipulator.types.ts

```diff
@@ -1,31 +1,72 @@
+// @needsAudit
 export type ImageResult = {
+  /**
+   * An URI to the modified image (usable as the source for an `Image` or `Video` element).
+   */
   uri: string;
+  /**
+   * Width of the image or video.
+   */
   width: number;
+  /**
+   * Height of the image or video.
+   */
   height: number;
+  /**
+   * it is included if the `base64` save option was truthy, and is a string containing the
+   * JPEG/PNG (depending on `format`) data of the image in Base64. Prepend that with `'data:image/xxx;base64,'`
+   * to get a data URI, which you can use as the source for an `Image` element for example
+   * (where `xxx` is `jpeg` or `png`).
+   */
   base64?: string;
 };
 
+// @needsAudit
+/**
+ * > If you specify only one value, the other will be calculated automatically to preserve image ratio.
```

Comment:
I'm wondering if it shouldn't be added to `resize` comment instead, what do you think?

Reply:
@tsapeta It was my first take, however the note blocks do not render that well in the tables right now, and omitting the block results in blend-in of the message with the field description. 

But probably, as you suggested, it's the better place for this note, and we can improve the notes styling in tables during upcoming docs redesign. 😄

Reply:
Honestly, it doesn't have to be a blockquote 😄 But I'm fine if only you'll remember to fix it in the future 😉

Reply:
CSS changes are required to fix this, so they will apply automatically, but maybe it the right call to just skip the blockquote 😄